codegen-compiler/src/main/kotlin/com/freeletics/khonshu/codegen/parser/ksp/CommonParser.kt

```diff
@@ -104,7 +105,11 @@ internal fun ClassName.stateMachineParameters(resolver: Resolver, logger: KSPLog
 internal fun KSClassDeclaration.findRendererFactory(logger: KSPLogger): ClassName? {
     val factory = declarations.filterIsInstance<KSClassDeclaration>()
         .firstNotNullOfOrNull {
-            if (it.allSuperTypes(false).any { superType -> superType == viewRendererFactory }) {
+            if (it.allSuperTypes(false).any { superType ->
+                    println("test: $superType")
```

Comment:
nit: do we still need this?

Reply:
Thanks for catching this, removed it [7d686d5](https://github.com/freeletics/khonshu/pull/594/commits/7d686d5dd8fe52202e30abc6a982bdd255282d26)